### ISSUE TITLE
How to preserve textures when cropping a mesh?

### ISSUE THREAD

**Issue by jedrekfulara** [mikedh/trimesh#1313]
I am using `m.slice_plane` to crop my mesh. My input mesh has textures, but the result of the crop has none. 
How can I preserve the textures on the cropped fragment?

**Comment by mikedh** [comment]
Yeah the slice operation currently doesn't have texture preservation implemented, PR's welcome! It seems like it would be relatively straightforward to implement, perhaps barycentric interpolation for the new UV coordinates along the slice plane, and just a copy of the original `visual.material`.

**Comment by jedrekfulara** [comment]
I'm very new to trimesh, so if you could give me some pointers on how to generate the new UV coordinates, I could give it a try.

**Comment by nepfaff** [comment]
This would be a great feature!
